amplify/backend/function/getParams.ts

```diff
@@ -62,3 +62,15 @@ export const getParams = async (params: Array<keyof typeof ParamNames>) => {
 
   return Promise.all(params.map((param) => getParam(param)));
 };
+
+export const setEnvVariables = async (
```

Comment:
Am I misunderstanding this? This method returns either the qa params array, or an empty array?

So why it called `set` instead of `get` ?

Reply:
this was a "copy and not think"™ about error, ive since refactored it so that it is much cleaner